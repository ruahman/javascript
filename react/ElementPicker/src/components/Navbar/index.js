import React from 'react';

import './navbar.css'

const Navbar = () => (
  <div className="navbar">
    <span className="navbar__label">Simple Website Builder</span>
  </div>
);

export default Navbar;
