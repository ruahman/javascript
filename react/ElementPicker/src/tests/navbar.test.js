import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Navbar from '../components/Navbar/index.js';

describe('Navbar', () => {
  it('works ok', () => {
    const component = shallow(<Navbar />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
