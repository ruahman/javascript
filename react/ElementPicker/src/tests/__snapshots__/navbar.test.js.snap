// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar works ok 1`] = `
<div
  className="navbar"
>
  <span
    className="navbar__label"
  >
    Simple Website Builder
  </span>
</div>
`;
